accepted/0000-registry-url-in-lock-file.md

```diff
@@ -0,0 +1,71 @@
+- Start Date: 2017-09-21
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The yarn.lock file includes information about the registry that it was generated against. It would
+be nice if there was no registry information associated with the lock file
+
+# Motivation
+
+We currently have two internal registries/mirrors. We generate the lock file against one we have in

```

Comment:
Add more details please.
Short examples how the yarn.lock files are different would be useful